### DISCUSSION TITLE
How to login with local account

### DISCUSSION THREAD

**Discussion by Ryckie** [requarks/wiki#4371] [Help / Questions]
Hello
I have hide local login, and bypass login screen enabled. (see image).
I now can no longer login with the local account. I tried adding "?all" to the url like so: (http://example.com:3000/login?all) but it keeps directing me directly to the discord auth page. How can I now login to the local account please?
![image](https://user-images.githubusercontent.com/63733138/131630838-3f7cc7c7-782c-41de-8906-acb55757ddf5.png)

**Comment by NGPixel** [comment]
Try `?all=1`. If that doesn't work, you could also try `?legacy=1`

**Reply by Ryckie** [reply]
Thank you. ?all did not work as it kept redirecting me to Discord auth. I went ahead and reinstalled the software. Didn't test legacy.

This may be marked as resolved.
